docs/book/modules/admin/pages/index.adoc

```diff
@@ -0,0 +1,42 @@
+////
+Module Type: CONCEPT
+////
+
+= Administration Guide
+
+This administration guide gives you a better understanding on Trustify's deployment options, installing the Trustify
+services locally, and reference information for Trustify's OpenAPI structures.
+
+Trustify relies on external services for storage and authentication.
+Trustify's services can use Amazon's S3 APIs to store object data.
+You do not need to run Trustify on Amazon Web Services (AWS), because other cloud vendors, such as Google Cloud Storage
+(GCS) and MinIO, implement Amazon's S3 API.
+
+It also is possible to run Trustify on bare metal. Either on a server, or a local desktop/laptop, for trying it out.
+
+== Deployment options
+
+You have several deployment options to choose from for running Trustify.
+
+=== Bare metal
+
+Running Trustify on bare-metal servers requires you to download the `trustd` binary from source found on the
+link:https://github.com/trustification/trustify/releases[Trustify releases] page.
+
+There currently exist two variants of this binary, `trustd` and `trustd-pm`. The `-pm` version includes a few embedded
+services (like PostgreSQL and an OIDC issuer) which make it easier to run Trustify locally for trying it out. However,
```

Comment:
For technical documentation, we should not use parenthetical statements.  Consider rewriting this, for example:

`... services (like PostgreSQL and an OIDC issuer) which make it ...` --> 
`... services, such as PostgreSQL database and an OpenID Connect (OIDC) issuer, which makes it ...`